packages/website/package.json

```diff
@@ -19,6 +19,7 @@
     "react-dom": "^18.2.0",
     "react-markdown": "^9.0.0",
     "react-router-dom": "^6.17.0",
+    "remark-gfm": "^4.0.0",
```

Comment:
peter & I discussed initial documentation, and the use case for this was removed. 

I think it's still worth keeping because I can imagine tables being useful. But we can also get rid of it until we actually use the functionality.